Slim/Plugin/FullTextSearch/Plugin.pm

```diff
@@ -287,7 +288,13 @@ sub createHelperTable {
 	$orderOrLimit = 'LIMIT 0' if !$tokens;
 
 	# The first 32 bytes of the ID are either an MD5 of the ID, or some buster to make it "non searchable" - remove that prefix
-	my $searchSQL = "CREATE $temp TABLE $name AS SELECT SUBSTR(fulltext.id, 33) AS id, FULLTEXTWEIGHT(matchinfo(fulltext)) AS fulltextweight FROM fulltext WHERE fulltext MATCH 'type:$type $tokens' $orderOrLimit";
+	my $searchSQL = "SELECT SUBSTR(fulltext.id, 33) AS id, FULLTEXTWEIGHT(matchinfo(fulltext)) AS fulltextweight FROM fulltext WHERE fulltext MATCH 'type:$type $tokens' $orderOrLimit";
+	if ($createPrimaryKey) {
+		$dbh->do("CREATE $temp TABLE $name (id integer primary key, fulltextweight integer)");
+		$searchSQL = "INSERT INTO $name $searchSQL";
+	} else {
+		$searchSQL = "CREATE $temp TABLE $name AS $searchSQL";
+	}
```

Comment:
I'd prefer if you did a separate `CREATE $temp` in all cases, so we can have the actual `$searchSQL` identical.